ZEP/instructions/zep0000.md

```diff
@@ -0,0 +1,227 @@
+
+> Standard Track ZEP
+
+Describes a new feature or implementation for Zarr
+
+> Informational ZEP
+
+Describes a ZEP design issue or provides general guidelines or information to the Zarr community but does not propose a new feature. Informational ZEPs do not 
+necessarily represent Zarr community consensus or recommendation, so users and implementers are free to ignore informational ZEPS or follow their advice.
+
+> Process ZEP
+
+Describes a new process around Zarr and its implementations, or proposes a change to(or an event in) a process. Process ZEPs are like Standard Track ZEPs but apply to areas other than the Zarr project itself. 
+They may propose an implementation, but not to Zarr’s codebase; they require community consensus; unlike informational ZEPs, they are more than recommendations, and users are typically not free to ignore them. 
+Examples include procedures, guidelines, changes to the decision-making process, and changes to the tools or environment used in Zarr’s development. Any meta-ZEP is also considered a Process ZEP.
+
+## ZEP Workflow
+
+The ZEP process begins with a new idea for Zarr. It is highly recommended that a single ZEP contain a single key proposal or new idea. Small enhancements or 
+patches often don’t need a ZEP and can be injected into the Zarr development workflow with a pull request to the Zarr ZEP repo. The more focused the ZEP, 
+the more successful it tends to be. If in doubt, split your ZEP into several well-focused ones.
+
+Each ZEP must have a champion—someone who writes the ZEP using the style and format described below, shepherds the discussions in the appropriate forums, 
+and attempts to build community consensus around the idea. The ZEP champion (a.k.a. Author) should first attempt to ascertain whether the idea is suitable for a ZEP. 
+Posting to the [Gitter](https://gitter.im/zarr-developers/community) is the best way to go about doing this.
```

Comment:
Given Gitter isn't very searchable would suggest an issue on the spec repo or discussion. Something that makes it easier to track the discussion. Ok if we want to share a link on Gitter. Just wanted to avoid making major decisions there

Reply:
Added a few more areas for discussion.